hack/generate-crd-deepcopy.sh

```diff
@@ -41,4 +41,4 @@ boilerplate > "${HEADER}"
 exec go run sigs.k8s.io/controller-tools/cmd/controller-gen \
     object \
     "object:headerFile=${HEADER}" \
-    paths=./apis/...
+    paths=./apis/projectcontour/...
```

Comment:
Is this change actually needed?

Reply:
We could maybe leave this one, it just updates the deepcopy things.